torchtitan/experiments/deepseek_v3/symm_mem_recipes/triton_on_device_all_to_all_v.py

```diff
@@ -156,3 +158,34 @@ def on_device_all_to_all_v(
     )
     # log_triton_kernel(kernel)
     return output
+
+
+class OnDeviceAllToAllV(torch.autograd.Function):
+    @staticmethod
+    def forward(
+        ctx,
+        output: torch.Tensor,
+        output_splits: torch.Tensor,
+        input: torch.Tensor,
+        input_splits: torch.Tensor,
+        group: dist.ProcessGroup = dist.group.WORLD,
```

Comment:
should we default this? it feels like usually you need to know what group you're doing this over. i could see leaving it as 'world' or unit testing purposes but for any real usage in models it seems unlikely the default would be sufficient?

Reply:
The default here is similar to torch.dist defaulting group to `None`